canonical-web-and-design/canonical.com
# Create "canonical" dropdown navigation item

## Done

- Replace existing top nav with Vanilla navigation
- Create the “Products” dropdown with the content already created for the same task([issue](https://github.com/canonical-web-and-design/web-squad/issues/4789), [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/11213)) on ubuntu.com 
- Load `navigation-products.html` from url to enable other sites to pull the content

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click `Products` in the nav and check the dropdown menu is displaying and working properly.
- Demo at https://canonical-com-496.demos.haus/
## Issue / Card

Fixes [#4847](https://github.com/canonical-web-and-design/web-squad/issues/4847) 

## Screenshots
<img width="1425" alt="Publisher of Ubuntu | Canonical ()" src="https://user-images.githubusercontent.com/57550290/157473431-ebc1ffcd-7505-477e-a19e-8b75798d70cd.png">
